src/components/ListLinks.js

```diff
@@ -0,0 +1,28 @@
+import style from 'styled-components';
+
+const ComponentListLinks = style.div`
+ display: flex;
+  margin-left: 1116px;
+  margin-top: -50px;
+  width: 100px;
+  grid-column-gap: 30px;
+  color: black;
+  font-size: 30px;
+`;
+const ListLinks = ({ className, id, type, children }) => {
+  return (
+    <ComponentListLinks
+      type={type ? type : 'button'}
```

Comment:
If this type is not used, please remove it

Reply:
Deleted component ListLinks.js